### ISSUE TITLE
optgroup compatibility

### ISSUE THREAD

**Issue by makromat** [JosHuybrighs/service.ddlist.jquery#1]
Hello,

Amazing script! But what about optgroup tag ? Is supported by your script ?

Thanks for answer

**Comment by JosHuybrighs** [comment]
No, optgroup tags are currently not supported. Not sure whether there is high demand for this. But I will have a look at it.

Jos

**Comment by makromat** [comment]
I think this is last missing feature in your script.
